client/src/main/scala/caliban/client/CalibanClientError.scala

```diff
@@ -3,7 +3,10 @@ package caliban.client
 /**
  * The base type for all Caliban Client errors.
  */
-sealed trait CalibanClientError extends Throwable with Product with Serializable
+sealed trait CalibanClientError extends Throwable with Product with Serializable {
+  def msg: String
+  override def getMessage: String = msg
```

Comment:
How about just `override def getMessage: String = toString`?

So we get the most details including for communication errors and decoding errors?

Reply:
Done